en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-configuration.md

```diff
@@ -0,0 +1,90 @@
+
+    2. Follow the online instructions.
+
+Part of the sign-up procedure involves receiving a phone call and entering a verification code using the phone keypad.AWS will notify you by email when your account is active and available for you to use.
+
+## Obtaining user credentials
+
+You can access the Amazon Lambda service using the root user credentials but these credentials allow full access to all resources in the account as you can't restrict permission for root user credentials.If you want to restrict certain resources and allow controlled access to AWS services then you can create IAM(Identity and Access Management) users in your AWS account in that case
+
+* **Follow the steps below to get an AWS Access Key for your AWS root account:**
+
+  1. Go to the AWS Management Console.
+  2. Hover over your company name in the right top menu and click "My Security Credentials".
+  3. Scroll to the "Access Keys" section.
+  4. Click on "Create New Access Key".
+  5. Copy both the Access Key ID (YOUR_AMAZON_LAMBDA_KEY) and Secret Access Key (YOUR_AMAZON_LAMBDA_SECRET).
+
+* **Follow the steps below to get an AWS Access Key for an IAM user account:**
+
+  1. Sign in to the AWS Management Console and open the IAM console.
+  2. In the navigation pane, choose Users.
+  3. Add a checkmark next to the name of the desired user, and then choose User Actions from the top.
+  4. Click on Manage Access Keys.
+  5. Click on Create Access Key.
+  6. Click on Show User Security Credentials. Copy and paste the Access Key ID and Secret Access Key values, or click on Download Credentials to download the credentials in a CSV (file).
+  
+## Create Amazon S3 Bucket
+
+  1. Navigate to the created **AWS** account.
+  2. Click **Services** tab on left top of the tab.
+  
+     <img src="/assets/img/connectors/amazon-services.png" title="Select amazon services" width="800" alt="Select amazon services"/>
+  
+  3. Select **Storage** section and click **S3**.
+  
+     <img src="/assets/img/connectors/amazon-s3-create-bucket-sample-1.png" title="Select S3 services" width="800" alt="Select S3 services"/>
+
+  4. Create a bucket.
+  
+     <img src="/assets/img/connectors/amazon-s3-create-bucket-sample-2.png" title="Create S3 bucket" width="800" alt="Create S3 bucket"/>
+       
+## Create Deployment Package
+
+  Your function's code consists of scripts or compiled programs and their dependencies. When you author functions in the Lambda console or a toolkit, the client creates a ZIP archive of your code called a [ deployment package ](https://docs.aws.amazon.com/lambda/latest/dg/deployment-package-v2.html). The client then sends the package to the Lambda service. When you manage functions with the Lambda API, command line tools, or SDKs, you create the deployment package. You also need to create a deployment package manually for compiled languages and to add dependencies to your function.
```

Comment:
`When you manage functions with the Lambda API,` this sentence seems incomplete.

Reply:
Removed this line. Linked with the AWS documentation, deployment package explanation page.